source/security.txt

```diff
@@ -131,51 +131,68 @@ Environment Variables
 ~~~~~~~~~~~~~~~~~~~~~
 
 The following code shows how to create a connection URI to
-authenticate with ``MONGODB-AWS`` when obtaining credentials from environment variables:
+authenticate with ``MONGODB-AWS`` when obtaining credentials from environment
+variables. Ensure you have your environment variables specified before running
+this code.
```

Comment:
The "Environment Variables" section seems superfluous after recent changes explicitly state the presence of related env vars. Recommend its removal.

Reply:
I agree that it does seem redundant given that it's the same as the code example above. However, this page is intended to be a way to quickly reference the syntax for a given mechanism. It's not intended to be read from top to bottom. Therefore, a reader that comes to the page and just jumps straight to the "Environment Variables" section will still need that code block. This also sticks with the conventions we have established on our other landing/usage example pages